kubernetes/generic/bookie.yaml

```diff
@@ -57,7 +57,7 @@ spec:
         spec:
             containers:
               - name: bookie
-                image: streamlio/pulsar:latest
+                image: apachepulsar/pulsar:latest
```

Comment:
(sent previous comment before getting your second one)

> I can see both latest/pulsar:1.19.0-incubating present at hub so, if we always push latest
> then it would be better to keep latest as it won't require code change.?

Yes the `latest` tag is special for Docker in that it's also the default tag that gets used when no version is specified. 

The script to publish the images makes sure to always push both tags to hub.docker.com :

https://github.com/apache/incubator-pulsar/blob/master/docker/publish.sh#L66